src/test/scala/io/pinecone/spark/pinecone/ParseCommonSchemaTest.scala

```diff
@@ -0,0 +1,59 @@
+package io.pinecone.spark.pinecone
+
+import org.apache.spark.sql.SparkSession
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should
+
+class ParseCommonSchemaTest extends AnyFlatSpec with should.Matchers {
+  private val spark: SparkSession = SparkSession.builder()
+    .appName("SchemaValidationTest")
+    .master("local[2]")
+    .getOrCreate()
+
+  private val inputFilePath = "/Users/rohan.s/Documents/sdk/spark-pinecone/src/test/resources"
```

Comment:
I think this may need to be updated? Looks like it's pointing at your local copy.

Reply:
Fixed!